WssElixir/app/src/main/java/com/example/wsselixir/presentation/home/HomeActivity.kt

```diff
@@ -0,0 +1,146 @@
+package com.example.wsselixir.presentation.home
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.ArrayAdapter
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.bumptech.glide.Glide
+import com.example.wsselixir.R
+import com.example.wsselixir.databinding.ActivityHomeBinding
+import com.example.wsselixir.databinding.DialogHomeBinding
+import com.example.wsselixir.presentation.home.adapter.FollowerAdapter
+import com.example.wsselixir.presentation.model.Follower
+import com.example.wsselixir.presentation.model.User
+import com.example.wsselixir.presentation.myInfo.MyInfoActivity
+
+class HomeActivity : AppCompatActivity() {
+    private lateinit var homeBinding: ActivityHomeBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        homeBinding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(homeBinding.root)
+
+        initMBTISpinner()
+        initPostButton()
+        initRecyclerView()
+    }
+
+    private fun initMBTISpinner() {
+        ArrayAdapter.createFromResource(
+            this,
+            R.array.mbti_array,
+            android.R.layout.simple_spinner_item
+        ).also { adapter ->
+            adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
+            homeBinding.spinnerHomeMBTI.adapter = adapter
+        }
+    }
+
+    private fun initPostButton() {
+        homeBinding.btnHomePost.setOnClickListener {
+            if (checkUserName() && checkUserMBTI()) {
+                makeToast("이름과 MBTI를 입력해주세요")
+            } else if (checkUserName()) {
+                makeToast("이름을 입력해주세요")
+            } else if (checkUserMBTI()) {
+                makeToast("MBTI를 선택해주세요")
+            } else {
+                navigateToMyInfoActivity()
```

Comment:
```suggestion
when {
    checkUserName() && checkUserMBTI() -> makeToast("이름과 MBTI를 입력해주세요")
    checkUserName() -> makeToast("이름을 입력해주세요")
    checkUserMBTI() -> makeToast("MBTI를 선택해주세요")
    else -> navigateToMyInfoActivity()
}
```
when 절을 사용해보시는 건 어떨까요 ? 가독성이 더 좋아질 듯 !

Reply:
매우 좋은것 같습니다 반영하겠습니다.